canonical-web-and-design/ubuntu.com
# Add resize notifications

## Done

- Add notifications on successful resizes
- Change the resize field to target `current_number_of_machines` instead of `number_of_machines`

## QA

- Go to [/advantage](https://ubuntu-com-11185.demos.haus/advantage?test_backend=true)
- log in with an account that has monthly subscriptions
- resize a monthly subscription up
- see that the dismissable notification now shows the number of machines you added
- resize a monthly subscription down, see that there is a permanent notification telling you when your subscription will be updated
- refresh the page and see that this notification still shows for the downsized subscription

## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/475 and https://github.com/canonical-web-and-design/commercial-squad/issues/476

## Screenshots

![image](https://user-images.githubusercontent.com/11927929/150949175-71aa4964-9a35-4759-b566-b2364b32c750.png)
